packages/loaders/json-schema/src/scalars.ts

```diff
@@ -28,3 +30,45 @@ export const ObjMapScalar = new GraphQLScalarType({
     return null;
   },
 });
+
+export const GraphQLCustomDateTime = new GraphQLScalarType({
+  name: 'CustomDateTime',
+  description:
+    'Custom DateTime scalar that wraps GraphQLDateTimeConfig and converts non-RFC 3339 strings to RFC 3339.',
+  serialize: value => {
+    // helper to check if the date-time string is RFC 3339 compliant
+    const validateDateTime = (dateTimeString: string) => {
+      dateTimeString =
+        dateTimeString === null || dateTimeString === void 0
+          ? void 0
+          : dateTimeString.toUpperCase();
+      const RFC_3339_REGEX =
```

Comment:
What is this regex exactly? Reference would be super helpful

Reply:
yep i will add a comment on this